schedule/easy_days.py

```diff
@@ -23,21 +25,20 @@ def easy_days(did):
     if not config.load_balance:
         tooltip("Please enable load balance first")
         return
-    if len(config.easy_days) == 0 and len(config.easy_dates) == 0:
+    if (
+        all([r == 1 for r in config.easy_days_review_ratio_list])
+        and len(config.easy_dates) == 0
+    ):
         tooltip("Please select easy days or specific dates first")
         return
     today = mw.col.sched.today
     due_days = []
-    if config.easy_days_review_ratio > 0 and config.auto_easy_days:
-        reschedule_range = 7
-    else:
-        reschedule_range = 35
-    for day_offset in range(reschedule_range):
+    for day_offset in range(35):
```

Comment:
Why was the code removed here? The previous code was aimed at improving performance when the user is using "Auto apply easy days on closing collection".

Reply:
Because `config.easy_days_review_ratio` has been removed, which slowed down the performance in the previous code. After refactor, the performance is improved, so I think it's OK to remove this condition.